docs/starting_server.md

```diff
@@ -1,24 +1,24 @@
 # Starting the Server  {#ovms_docs_serving_model}
 
 There are two method for passing to the model server information about the models and their configuration:
-- via CLI parameters - for a single model 
+- via CLI parameters - for a single model or pipeline
 - via config file in json format - for any number of models and pipelines
 
 Note that changing configuration in runtime while serving is possible only with the config file.
 When deploying model(s) with a configuration file, you can add or delete models, as well as update their configurations in runtime, without needing to restart the server.
 
-## Serving a Single Model
+## Serving a Classic Model
```

Comment:
use a common category for preconfigured single classic model and single pipeline.
Those steps are the same.
Second option would be pulling genai models during the launch
The last would be for multi model/pipeline deployment with config.json including managing this file via cli